src/main/java/team/creative/creativecore/common/gui/GuiParent.java

```diff
@@ -42,17 +39,21 @@ public class GuiParent extends GuiControl implements IGuiParent, Iterable<GuiChi
     
     public GuiParent(String name, GuiFlow flow) {
         super(name);
-        this.flow = flow;
+        this.setFlow(flow);
     }
     
     public GuiParent(String name, GuiFlow flow, VAlign valign) {
         this(name, flow, Align.LEFT, valign);
     }
+
+    public GuiParent(String name, GuiFlow flow, Align align) {
+        this(name, flow, align, VAlign.TOP);
+    }
     
     public GuiParent(String name, GuiFlow flow, Align align, VAlign valign) {
         this(name, flow);
-        this.align = align;
```

Comment:
Same here and I don't see a reason to use the methods.

Reply:
Logic reduction 
Example: if do a custom GuiParent and I want to place custom states based on any original property of GuiParent I can just override setAlign and place the logic there and avoid copy the same logic or call the method in the constructor 

I forgot to move into the constructor the default values with the proper method call but I will do that in another PR

Reply:
But the field is public anyway, meaning this will not work properly in the current state. So I rather want to revert that back.

Reply:
reverted